### DISCUSSION TITLE
Setup the debug service on IBM i - Setup Certificates step  getting message  "Failed to create certificates"

### DISCUSSION THREAD

**Discussion by fabiokang** [codefori/vscode-ibmi#1754] [IBM i] [answered]
Hi , I am trying to setup the debug service on IBM i and the step Setup Certificates - Generate certificates , I am getting message "Failed to create certificates", there is not much more detail.

On the IBM i  I noticed that /qibm/ProdData/IBMiDebugService/bin/certs  is created but nothing inside.

I looked the QP0ZSPWP joblogs ,but no messages related to this problem.

I successfully connect to the system and edit my program.

I will appreciate any suggestions.

Thank you. Fabio

**Comment by worksofliam** [comment]
Please share the contents of your Code for IBM i output channel after it fails.

<img width="939" alt="image" src="https://user-images.githubusercontent.com/3708366/178739532-03c8ad30-84de-4698-9904-65938f2ef651.png">

**Reply by fabiokang** [reply]
Hi, I attached the output . Thank you

[code for i output Failed to create certificates.txt](https://github.com/codefori/vscode-ibmi/files/13812739/code.for.i.output.Failed.to.create.certificates.txt)

**Comment by fabiokang** [comment, marked as answer]
Hi , I tried using a different profile and I notice I got the message "IBM recommends using bash as your default shell",  which I set, and after disconnecting / reconnecting, I was able to generated the Certificates.
